src/integrationTest/java/uk/gov/hmcts/reform/cwrdapi/config/TestConfig.java

```diff
@@ -52,4 +55,15 @@ public void populateCaseWorkerProfile(CaseWorkersProfileCreationRequest cwrdProf
             }
         };
     }
+
+    @Bean
+    @Primary
+    public HttpClient testClient() {
+        return HttpClientBuilder.create().setRetryHandler((exception, executionCount, context) -> {
```

Comment:
1. Override the default httpclient and use Apache HttpClient.
2. Added retry logic for wiremocks.

Reply:
not sure why we need this?

Reply:
Hi Abhijit, our integration test cases are running quite fast and sometimes the wiremock server is not able to take up requests that quickly. Hence added a retry logic so that wiremock server gets a chance to perform stubbing properly.